Classes/Plugin.php

```diff
@@ -34,8 +34,14 @@ public function on($eventKey, $data = null) {
 			// limit words function -- very rough limit due to HTML input
 			$limit_words = new \Twig_SimpleFilter('limit_words', function ($string) {
 				$words = str_word_count($string, 2);
+				$nbwords = str_word_count($string, 0);
 				$pos = array_keys($words);
-				return trim(substr($string, 0, $pos[$this->settings['limit']])) . '...';
+				if ($this->settings['limit'] > $nbwords) {
```

Comment:
Can you change this into `>=`?

Having the word-count exactly the same length as the limit throws a error too. This would cover that case.